src/libbson/src/jsonsl/jsonsl.c

```diff
@@ -75,7 +75,6 @@ void jsonsl_dump_global_metrics(void)
 #else
 #define INCR_METRIC(m)
 #define INCR_GENERIC(c)
```

Comment:
`INCR_STRINGY_CATCH` is also conditionally defined above. Consider removing the entire: `#ifdef JSONSL_USE_METRICS` block (I expect is used).

Reply:
Removed all unnecessary and unused code in jsonsl conditioned on macros which we never define or use.